src/tests/ftest/util/server_utils.py

```diff
@@ -184,6 +184,20 @@ def using_dcpm(self):
             value = self.yaml.using_dcpm
         return value
 
+    def kill(self):
+        """Forcibly terminate any server process running on hosts."""
+        regex = self.manager.job.command_regex
+	"""Try to dump all server's ULTs stacks before kill."""
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 195)